src/backends/tensorflow/base_backend.cc

```diff
@@ -143,8 +148,14 @@ BaseBackend::CreateExecutionContext(
                             ? Config().default_model_filename()
                             : cc_itr->second;
 
+    // Get virtual device tracker instance, and get next device id
+    if (!graphdef_backend_config->memory_limit_mb.empty()) {
```

Comment:
Create VirtualDeviceTracker::HasVirtualDevice() and use that to determine if vgpus should be used